winterfell/models/userValues.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+var mongoose = require('mongoose');
+var User = require('./user');
+var Schema = mongoose.Schema;
+
+var UserValueSchema = new Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
+  },
+  values: Schema.Types.Mixed
+}, { timestamps: true });
```

Comment:
Agree everything in the javascript world should be camel case, thats kind of the convention everyone follows. Mongo is technically JSON-ish so using camek case in there makes sense.